packages/cdktf-cli/test/lib/cdktf-project.test.ts

```diff
@@ -6,13 +6,24 @@ import * as os from "os";
 import { CdktfProject, get, init, Language } from "../../lib/index";
 import { SynthesizedStack } from "../../lib/synth-stack";
 import { getMultipleStacks } from "../../lib/cdktf-project";
+import { exec } from "cdktf-cli/lib/util";
 
 function eventNames(events: any[]) {
   return events
     .map((event) => event.type)
     .filter((name) => !name.includes("update"));
 }
 
+jest.mock("cdktf-cli/lib/util", () => {
```

Comment:
It's been a while since I've done this, so maybe there's a better way to do this?

Reply:
Seems to be the way to go with Jest 😁 I don't know a better one 🤷‍♂️